### ISSUE TITLE
Bug: Repulsion Gel does not work on rotating panels

### ISSUE THREAD

**Issue by Wolfcl0ck** [StrataSource/Portal-2-Community-Edition#1651]
### Build Information
**Chaos.exe Build Date:** 06-11-2023 22:13:42 UTC
**Server Build Date:** 06-11-2023 22:24:40 UTC
**Client Build Date:** 06-11-2023 22:24:41 UTC
**Branch:** Default, using Panorama UI

### Hardware Information
**CPU:** AMD Ryzen 9 3900X 12-Core Processor 3.80 GHz
**GPU:** 4095MB NVIDIA GeForce RTX 2070 (EVGA)
**RAM:** 32 GB DDR4 RAM, 2x 16 GB Corsair

### Bug Information
Placing Repulsion Gel on dynamic rotating panels makes the gel not function. Was able to reproduce between multiple launches of the game.

### Additional Potential Factors
"cl_portal_crosshair_mode" was set to 1
Game is running in Borderless Windowed mode
Game is running with -multirun enabled
Game is running with -insecure enabled

### Footage

https://github.com/StrataSource/Portal-2-Community-Edition/assets/16748747/4c8ddb2c-ed25-4bc0-82ce-8a9408464ec3

### Issue Map

sp_a3_speed_flings

### To Reproduce

1. Open sp_a3_speed_flings
2. Attempt to solve as usual
3. the gel on the rotating panel will fail

### Operating System

Windows 10 Home 64-bit, build 19045.3570

**Comment by CnicKTheCrocodile** [comment]
yeah that happened too and that's why I'd like to make the report

**Comment by TeamSpen210** [comment]
Seems to be fixed now.
